source/policies/deprecation.rst

```diff
@@ -55,44 +55,12 @@ Then, if your code directly depends on ``doctrine/orm`` which provides the
         }
     }
 
-Using ``trigger_error()`` and ``phpunit-bridge``
-------------------------------------------------
+Using ``doctrine/deprecations``
+-------------------------------
 
-In some cases, you may need to conditionally deprecate functionality. In these cases, it is required
-to use the ``trigger_error()`` function and a tool like `PHPUnit Bridge`_ which implements a custom
-error handler for you to use to collect and report these warnings. **This method is not recommended
-and Doctrine will try to avoid using it if possible.**
```

Comment:
Note: here I'm removing the sentence about this method being not recommended, because I don't see any issue with it when using a `psr/logger`, but I'm afraid that might be a personal opinion. I think at least @Ocramius has things to say on the matter, but it's unclear to me what. See https://github.com/doctrine/persistence/pull/281#issuecomment-1101638803

I read https://github.com/doctrine/doctrine-website/issues/200, but it doesn't mention `doctrine/deprecations`.

Personally, I'm not sure trying hard to go the `@deprecated` way every time is the way to go: the implication is that you have to come up with new names for classes/methods/etc. in favor of which to deprecate existing counterparts. This could result in an awkward API because of bad naming.